src/Dialog.elm

```diff
@@ -136,10 +136,18 @@ wrapFooter footer =
 
 backdrop : Maybe (Config msg) -> Html msg
 backdrop config =
-    div [ classList [ ( "modal-backdrop in", isJust config ) ] ]
+    div ((::) (classList [ ( "modal-backdrop in", isJust config ) ]) (backdropCloser config))
```

Comment:
This feels a bit hard to read, do you think you could simplify it a bit? I think I actually preferred the previous way you did it

Reply:
Yeah, it's not the nicest. Reverted to the similar approach